staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -29,6 +29,7 @@ const (
 	DeprecatedRollbackTo           = "deprecated.deployment.rollback.to"
 	DeprecatedTemplateGeneration   = "deprecated.daemonset.template.generation"
 	StatefulSetPodNameLabel        = "statefulset.kubernetes.io/pod-name"
+	StatefulSetPodOrdinalLabel     = "statefulset.kubernetes.io/pod-ordinal"
```

Comment:
Please add good description for this label, and also add godoc to the v1 StatefulSet struct that describes the labels created on each pod in the set.

Reply:
Added documentation:

![screen shot 2018-09-20 at 3 06 11 pm](https://user-images.githubusercontent.com/6752382/45841182-0577ba80-bce7-11e8-9813-7188d12e62dc.png)

Very open to feedback on wording here...